fga-eps-mds/2020.1-stay-safe-front-end
# Create Occurrence Register Screen

## Issue #
#9 

## Description
Created occurrence register screen with all necessary validations and conection with user-service api.

## Checklist:



## Known problems:
Occurrence location is static yet because need to be set in another screen.

The theme of DateTimePicker is editable only with native code.

## How to validate?
Use LAN connection on Expo to make a request to the service.

The user needs to be logged in to enter this screen, but the previous screen is not made yet, so make sure to be logged in to register an occurrence, you can make this changing the routes of the app and adding a navigation to occurrence screen after login/register of an user.
